website/docs/d/availability_zones.html.markdown

```diff
@@ -38,21 +40,44 @@ resource "aws_subnet" "secondary" {
 }
 ```
 
+### By Filter
+
+```hcl
```

Comment:
Does this look okay? https://github.com/terraform-providers/terraform-provider-aws/pull/12400/commits/6a7714f6806dba747e92a1a5a39d020c470b4f3d

Reply:
👍

Reply:
Though it might also be good to mention in the main text above something like

> When Local Zones are enabled in a region, by default this data source includes Local Zones and Availability Zones. To return only Availability Zones, see the example section below.

Reply:
I love it, I'll add that above and link the Local Zones text to the AWS docs.